Bottom: 384ca1d08c5a16b2c731c7c9c507b51f977a136b
Top:    2d4c256dfd1c1162d6ed0da5c3604b45fb62878d
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-14 11:10:49 +0800

Refresh of refresh-temp

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index f83680a..bcdb3ba 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -790,12 +790,12 @@ static void handle_tx_zerocopy(struct vhost_net *net)
 		}
 
 		total_len += len;
-
 		if (total_len < VHOST_NET_WEIGHT &&
 		    vhost_has_more_pkts(net, vq)) {
 			msg.msg_flags |= MSG_MORE;
 		} else {
 			msg.msg_flags &= ~MSG_MORE;
+		}
 
 		/* TODO: Check specific error and bomb out unless ENOBUFS? */
 		err = sock->ops->sendmsg(sock, &msg, len);
